SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj

```diff
@@ -1,159 +1,31 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{15E3FC9B-7714-4468-A296-81627A5B8F01}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.AppCenter</RootNamespace>
+    <TargetFrameworks>uap10.0.16299;net461</TargetFrameworks>
```

Comment:
BTW this way requires NuGet 4.4 or above

Reply:
Do we need to update something? Is this a breaking change?

Reply:
I see only one mention in public docs. + I had to update NuGet CLI locally (I had installed 4.3 to test minimal supported version).

So, it's not breaking change and only one number to update.

Reply:
But if the docs for 3.0 says minimal nuget version is 4.3 and now we bump to 4.4, isn't that a breaking change once we release 3.1?

Reply:
4.4 is required only for building (not using) SDK, so it is not a breaking change. Also, we've already bumped to 3.1 anyway

Reply:
Did we test building nugets with this PR then consuming with 4.3? I think we should before closing on this.

Reply:
Ok got confirmation on direct message.